lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -3803,13 +3803,15 @@ namespace Js
         case AsmJsRetType::Uint32x4:
         case AsmJsRetType::Uint16x8:
         case AsmJsRetType::Uint8x16:
+#if _WIN32 || _WIN64 //WASM.SIMD ToDo: Enable thunk for Xplat
```

Comment:
Not sure if PAL was defining `_WIN64`.. If all we want is to rule `xplat` out, can we use `#ifdef _WIN32` instead?

Reply:
Yes. This change is incorporated here
https://github.com/Microsoft/ChakraCore/pull/4074/files#diff-c1e1755ec9c1d66c55cceabcbdb92bc2R3806

I didnt know this earlier. Thanks for pointing it out!